vite.config.js

```diff
@@ -24,7 +24,6 @@ export default defineConfig({
 			input: allHtmlEntries,
 		},
 	},
-	base: '/codebeamer-miro/',
```

Comment:
Please revert that.  
This is a local setting, which would make the productive deployment unreachable.  

Locally, we're serving from the root, but the github page is `https://codebeamer-extensions-and-addons.github.io/codebeamer-miro/`. The subpath must be indicated to `vite`.

Reply:
I didn't change anything here. I'm confused